LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Learning@home authors and collaborators
+Copyright (c) 2024 Learning@home authors and collaborators
```

Comment:
Let's keep the scope of changes limited to the purpose of the PR, please. All other changes can be addressed in parallel with smaller pull requests

Reply:
I changed this so that tests would re-run, last night. Reverted.